app/forms/assessor_interface/verify_references_form.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+class AssessorInterface::VerifyReferencesForm
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+
+  attr_accessor :assessment
+  attribute :verify_references, :boolean
+  validates :verify_references, inclusion: [true, false]
+
+  delegate :application_form, to: :assessment
+
+  def save
+    return false if invalid?
+
+    assessment.update!(references_verified:)
```

Comment:
```suggestion
    assessment.update!(references_verified: verify_references)
```

I think the field in the form doesn't match the field in the database, although maybe we should rename the one in the form?

Reply:
Ah! :man_facepalming: I've amended to use the AR field name throughout.